docs/fast_data/configuration/single_view_creator.md

```diff
@@ -149,9 +149,17 @@ service.addHook('onClose', async() => {
 | KAFKA_CONSUMER_MAX_WAIT_TIME_MS     | -        | (v6.2.1 or higher) The maximum amount of time in milliseconds the server will block before answering the fetch request if there isn't sufficient data to immediately satisfy the requirement given by minBytes [1 byte]                                                                                                                                                                                                                                                                                                     | 500                 |
 | SV_UPDATE_VERSION                   | -        | (v6.2.1 or higher) Define which version of the `sv-update` event should be emitted by the service. Accepted values are `v1.0.0` and `v2.0.0`. By default, for retro-compatibility, version `v1.0.0` is employed                                                                                                                                                                                                                                                                                                             | v1.0.0              |
+| CONTROL_PLANE_ACTIONS_TOPIC         | -        | Topic where the Single View Creator expects to receive the action commands of `pause` and `resume`.                                                                                                                                                                                                                                                                                                                                                                                                                         | -                   |
+| CONTROL_PLANE_KAFKA_GROUP_ID        | -        | The Kafka Consumer Group Identifier for the action commands client.                                                                                                                                                                                                                                                                                                                                                                                                                                                         | -                   |
 
 If you do not want to use Kafka in the Single View Creator, you can just not set the environment variable *KAFKA_CLIENT_ID* or *KAFKA_BROKERS*. If one of them is missing, Kafka will not be configured by the service (requires *single-view-creator-lib* `v9.1.0` or higher)
 
+:::note
+
+If you want to enable the communication between the Single View Creator and the [Runtime Management](/fast_data/runtime_management.mdx) the variables `CONTROL_PLANE_ACTIONS_TOPIC`, `CONTROL_PLANE_KAFKA_GROUP_ID`, and `READ_TOPIC_FROM_BEGINNING` must be set.
```

Comment:
Do we really need `READ_TOPIC_FROM_BEGINNING` variable?

Reply:
No, we don't. It was mistakingly left there during  development.